paimon-flink/paimon-flink-common/src/test/java/org/apache/paimon/flink/SchemaChangeITCase.java

```diff
@@ -685,7 +684,6 @@ public void testModifyNullability() {
                                         "INSERT INTO T VALUES('aaa', 'bbb', 'ccc', 1, CAST(NULL AS FLOAT))"))
                 .satisfies(
                         anyCauseMatches(
-                                TableException.class,
```

Comment:
Will the introduction of Flink 2.1 break this exception class?

Reply:
Yes, Flink 2.1 changes the exception type from `TableException` to `EnforcerException`. Instead of making the test to adapt to both cases, I think it is easier to just ignore the type and just make sure the exception message is correct.

see https://github.com/apache/flink/pull/26540